struct.go

```diff
@@ -220,6 +220,18 @@ func (s *Struct) InsertInto(table string, value ...interface{}) *InsertBuilder {
 	return s.InsertIntoForTag(table, "", value...)
 }
 
+//`InsertIgnoreInto` is the Same with `InsertInto`, except for inserting with the existing primary key or unique key.
+//When inserting with the existing primary key or unique key, the db would ignore this operation with no error.
```

Comment:
Some wording issues. All newly added docs have same issues.

* Leave a blank after every `//`.
* The comment should start with a definition of this func, e.g. ReplaceIntoForTag creates a new `InsertBuilder` for `REPLACE` with table name. We need to provide enough information to doc readers. We should not force them to check another func to understand how to use this func.
* `the Same with` -> `similar to` (and mind the case please)
*  The sentence `except for ...` is not true. The responsibility of this package is only to build SQL without any execution. We need to tell readers what would happen to the SQL we build, e.g. using `INSERT IGNORE INTO` instead of `INSERT INTO`.
*  As of previous review comment, the sentence `When inserting with ...` should be deleted. I believe package users are DB experts and know how to use `INSERT IGNORE INTO`.

You can communicate with me for doc in person before submitting a new version. It would be more effective for us.